testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.IO;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+#endif
+using UnityEngine;
+
+/// <summary>
+/// This is needed as Unity throws "An abnormal situation has occurred: the PlayerLoop internal function has been called recursively. Please contact Customer Support with a sample project so that we can reproduce the problem and troubleshoot it."
+/// when trying to build from Setup() steps in tests.
+/// </summary>
+public static class BuildMultiprocessTestPlayer
+{
+    public const string MultiprocessBaseMenuName = "MLAPI Multiprocess Test";
```

Comment:
I think we could collapse menus under top-level "MLAPI" since we'd potentially reuse MLAPI/XXX sub-menus for other things later
```suggestion
    public const string MultiprocessBaseMenuName = "MLAPI/Multiprocess Test";
```